include/9on12draw.inl

```diff
@@ -47,6 +47,11 @@ namespace D3D9on12
     {
         // Data uploaded for a draw is only valid for that draw, and should be cleaned up immediately after to prevent
         // successive draws from reading stale/deleted data
+        if (device.GetPipelineState().GetIntzRestoreZWrite())
+        {
+            device.GetPipelineState().GetPixelStage().SetDepthStencilState(device, D3DRS_ZWRITEENABLE, 1);

```

Comment:
I'm not sure this is right. What if an app does the following sequence:
* Bind an `INTZ` texture for both input and output and draw with depth enabled.
* Disable `ZWRITEENABLE`
* Unbind the `INTZ` texture.
* Draw again.

This would override the app's request to disable `ZWRITEENABLE` because we'd still have `m_intzRestoreZWrite` set to true from the first draw. I think we need to move the logic from `SetDepthStencilState` for `ZWRITEENABLE` into a location that runs at draw time.

Reply:
Ah, I missed that this was the epilogue, I was thinking this was in the prologue. This is probably okay then as long as we ensure that the epilogue is always invoked in `SetIntzRestoreZWrite(true)` might've happened.